examples/release_highlights/plot_release_highlights_1_5_0.py

```diff
@@ -0,0 +1,180 @@
+# ruff: noqa
+"""
+=======================================
+Release Highlights for scikit-learn 1.5
+=======================================
+
+.. currentmodule:: sklearn
+
+We are pleased to announce the release of scikit-learn 1.5! Many bug fixes
+and improvements were added, as well as some new key features. We detail
```

Comment:
```suggestion
and improvements were added, as well as some key new features. We detail
```
I can't explain why, but I am pretty sure this is the right order. It is like a "big red lorry", not "red big lorry" - I have no idea why but the first is right and the second is wrong :D

Reply:
So we've been making this mistake since 0.23 :smile: